homeassistant/components/advantage_air/entity.py

```diff
@@ -36,19 +36,19 @@ def __init__(self, instance, ac_key):
         )
 
     @property
-    def _ac(self):
+    def _ac(self) -> dict:
```

Comment:
Can the keys be something other than strings?
```suggestion
    def _ac(self) -> dict[str, Any]:
```